swim/update_rollup_test.go

```diff
@@ -101,7 +103,7 @@ func (s *RollupTestSuite) TestFlushesAfterInterval() {
 	s.NotNil(s.r.FlushTimer(), "expected timer to be set")
 	s.Len(s.r.Buffer(), 2, "expected two updates to be in buffer")
 
-	time.Sleep(2 * time.Millisecond)
+	s.node.clock.(*clock.Mock).Add(2 * time.Millisecond)
```

Comment:
Agreed on both counts. IMO, Sleep should be an alias for Add in this case. But I have no intention of updating the lib right now for this case.